tools/sst_dump_test.cc

```diff
@@ -305,6 +305,28 @@ TEST_F(SSTDumpToolTest, NoSstFile) {
   }
 }
 
+TEST_F(SSTDumpToolTest, CheckValidSSTPath) {
+  Options opts;
+  opts.env = env();
+  std::string file_path = MakeFilePath("rocksdb_sst_test.sst");
+
+  char* usage[3];
+  PopulateCommandArgs(file_path, "--command=verify", usage);
```

Comment:
Right now you test
{--command=verify} * {sst file doesn't exist, sst file is valid}

I'd like that expanded to
{--command=verify, --command=identify} * {sst file doesn't exist, file contains something plain text, sst file is valid}

Reply:
Will add the tests correspondingly :)